src/libopensc/reader-pcsc.c

```diff
@@ -1314,8 +1335,8 @@ static void detect_reader_features(sc_reader_t *reader, SCARDHANDLE card_handle)
 }
 
 int pcsc_add_reader(sc_context_t *ctx,
-	   	char *reader_name, size_t reader_name_len,
-		sc_reader_t **out_reader)
+	char *reader_name, size_t reader_name_len,
+	sc_reader_t **out_reader)
```

Comment:
the continuation should be two tabs. See #2017 for more information.

Reply:
Code formatting in reader-pcsc.c is already rather inconsistent before this PR. For example `if(` vs `if (` feels like 50:50. :smiley: